google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -87,7 +86,7 @@ public void trackEvent(@NotNull String eventCategory,
 
         List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
         postData.add(new BasicNameValuePair("tid", analyticsId));
-        postData.add(new BasicNameValuePair("cd19", externalPluginName));  // Event type
+        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
```

Comment:
Ah, because of the no-op implementation?

In any case, we then need to make sure that `virtualPageUrl` below is

`"/virtual/" + eventCategory + "/" + eventAction`